### ISSUE TITLE
Can't launch app installer inside TrebleShot

### ISSUE THREAD

**Issue by adbenitez** [genonbeta/TrebleShot#72]
**Describe the bug**
When I receive an apk I am not able to launch (inside TrebleShot) the app installer view by touching the apk file, other file formats launch the associated application. (tested on 3 devices, same problem)

**To Reproduce**
1. share an apk from device 1 to device 2
2. on device 2 go to the file explorer tab in the app's menu.
3. locate the received apk and try to launch the installer by touching it.
4. See an empty list with title "Open with" and  a message "No apps can perform this action." in the list's body.

**Expected behavior**
The apk installer is launched when I touch an apk file.

**Environment (please complete the following information):**
 - Android version: 6.0, and others that I don't have the info right now
 - Language: English
 - App version: 1.2.7.7
 - Device Model and Brand: BLU Advance 4.0M, and others that I don't have the info right now

**Comment by velitasali** [comment]
This issue only occurs on Android 6 and 6.0.1 somehow. Other Android versions just work okay. I have tried different approaches but it was breaking others.

**Comment by adbenitez** [comment]
You are right, it works in 7.0, can't you simple ask for android versions and use the method that you have tried if it is android 6 and 6.0.1 and use the regular method otherwise??? something similar to this:
https://stackoverflow.com/questions/3423754/retrieving-android-api-version-programmatically

**Comment by velitasali** [comment]
> You are right, it works in 7.0, can't you simple ask for android versions and use the method that you have tried if it is android 6 or 6.0.1 and use the regular method otherwise??? something similar to this:
> https://stackoverflow.com/questions/3423754/retrieving-android-api-version-programmatically

Clearly, but Android 6 is resisting on not opening the APK files in other words I was not able to come with a solution that does not mess things up at the same time. 

This is the code that produces the intent and this is not changed regarding the OS executing it. It is no platform specific code, but only breaks APK files on Android 6. 
https://github.com/genonbeta/framework-android/blob/361f5942610aa89680df30bd57857a4b44c61c33/framework/src/main/java/com/genonbeta/android/framework/util/FileUtils.java#L36

**Comment by adbenitez** [comment]
Applications like Ghost Commander, AnExplorer, F-Droid client, etc. do this on Android 6, perhaps a quick look at they code can give you some hint on the problem or at least an alternative solution that works for all Android versions:
https://github.com/PDi-Communication-Systems-Inc/ghostcommander
https://github.com/1hakr/AnExplorer
https://github.com/f-droid/fdroidclient

**Comment by velitasali** [comment]
OK, I have just discovered that Marshmallow and below Android versions' package installer doesn't support content:// scheme which the app heavily relies on. Since everything was converted to work with content providers I cannot assume the file path, so I guess I should bring some old code back. This is why the only thing that was broken was apk files.
